docs/triaging-an-issue.md

```diff
@@ -0,0 +1,44 @@
+# Maintainers: Triaging an Issue
+
+**This guide is for maintainers.** These special people have **write access** to one or more of Jekyll's repositories and help merge the contributions of others. You may find what is written here interesting, but it’s definitely not for everyone.
+
+Here are some key things to remember when evaluating an issue.
```

Comment:
@beardofedu the ticket template takes care of that in asking the user within the first few lines to let us know if this is a feature request or otherwise.  Sometimes it's hard for a user to tell if something is a feature or a bug, a fundamental concept left out is a bug to some but a feature to others.  Solving an edge might be a feature to us but a bug to them.